supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -164,6 +164,51 @@
 
 *See also*:
 
+[discrete]
+[[operator]]
+==== image:images/yes.png[yes] Operator (noun)
+*Description*: In the context of Kubernetes, an Operator is a method of packaging, deploying, and managing a
+Kubernetes application. A Kubernetes application is an application that is both
+deployed on a Kubernetes cluster (including OpenShift clusters) and managed
+using the Kubernetes APIs and `kubectl` or `oc` tooling.
+
+The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators. For example:
+
+----
+= Support policy for unmanaged Operators
+
+Individual Operators have a `managementState` parameter in their configuration.
+----
+
+----
+The following operators and operands are supported in Drools Rule Language:
+
+* + (addition)
+* - (subtraction)
+...
+----
+
+An Operator's full name must be a proper noun, with each word initially
+capitalized. If it includes a product name, defer the product's capitalization
+style guidelines. For example:
+
+- Red Hat OpenShift Logging Operator
+- Prometheus Operator
+- etcd Operator
+- Node Tuning Operator
+- Cluster Version Operator
+
+While "containerized" is allowed, do not use "Operatorize" to refer to building
+an Operator that packages an application.
+
+NOTE: When referring generally to other Kubernetes components, such as pods, nodes, or image streams, use lowercase. When referring to a specific component, follow the capitalization of the component name and apply monospace formatting, such as "the `Pod` spec", "a `Node` object", or "an `ImageStream` resource".
+
+*Use it*: yes
+
+*Incorrect forms*: operator, operatorize
+
+*See also*: link:https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting[API object formatting] in OpenShift documentation guidelines
```

Comment:
Instead of pointing to our GitHub repo, I'd just use the entry we have on this already in this guide:

```
*See also*: xref:api-objects[API objects]
```

Reply:
Sure, I just thought the OpenShift doc guidelines provided a bit more helpful info, but fine by me. Done.

Reply:
Actually, on second thought, I recall now the other reason I referred to the OpenShift docs repo. This and the new xref I've just added to the OpenShift section per your suggestion won't work because they're xref's outside of the file. So it'll need to just be the repo link to the relevant section like I had, unless we want to get fancier with symlinking or a different way to xref outside of the file. I think the repo+section anchor link might be easiest. Whatever you prefer.

Reply:
@sterobin Can you clarify how the xref won't work? If you look at the live version of the SSG, it's working right now.

For example: https://redhat-documentation.github.io/supplementary-style-guide/#_node_noun

That is in the fuse.adoc file, but xrefs to a term in the general conventions n.adoc file.

Reply:
@bergerhoffer Ah, okay, it's just not working locally then, and not in github. So if we're okay with this being functional only in the final ouptut but not in the repo preview etc., fine by me.

Reply:
Right, it wouldn't work from GitHub. But you should be able to test it locally by doing `asciidoctor master.adoc` and viewing the generated `master.html` file.